feature.env

```diff
@@ -49,6 +49,7 @@ COMPONENT_API_URL=http://localhost:9091/component
 PERSON_INTEGRATION_API_URL=http://localhost:9091/personIntegration
 CSIP_API_URL=http://localhost:9091/csip
 HEALTH_AND_MEDICATION_API_URL=http://localhost:9091/healthAndMedication
+OS_PLACES_API_URL=http://localhost:9091/search/places/v1
```

Comment:
`localhost:9091` is wiremock and we tend to prefix the URL with the API name to avoid conflicts (i.e. many APIs could have a `GET /prisoner/{prisonerNumber}` endpoint) so could this be `http://localhost:9091/osPlaces/search/places/v1`?

Reply:
Done :)